integrations/asana/.nango/schema.ts

```
@@ -1,148 +1,140 @@
 // ---------------------------
-// This file was generated by Nango (v0.62.0)
+// This file was generated by Nango (v0.64.1)
 // You can version this file
 // ---------------------------
 
-export interface Id {
-    id: string;
-}
+export interface SyncMetadata_asana_tasks {
+};
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)